contrib/test/integration/vars.yml

```diff
@@ -11,3 +11,19 @@ crio_integration_filepath: "{{ artifacts }}/testout.txt"
 crio_node_e2e_filepath: "{{ artifacts }}/junit_01.xml"
 result_dest_basedir: '{{ lookup("env","WORKSPACE") |
                          default(playbook_dir, True) }}/artifacts'
+
+# The shell command to execute the integration tests
+int_test_cmd: "make localintegration >& {{ crio_integration_filepath }}"
+
+# Environment variables to set when executing int_test_cmd
+int_test_env:
+    CGROUP_MANAGER: "cgroupfs"
+    STORAGE_OPTIONS: >
+        --storage-driver=overlay
+        {% if ansible_distribution in ['RedHat', 'CentOS'] %}
+            --storage-opt overlay.override_kernel_check=1
+        {% endif %}
+
+# Additional environment variables for int_test_cmd with usernamespaces
+userns_int_test_env:
+    TEST_USERNS: "1"
```

Comment:
What does this do? Who/where relies on this? `ack USERNS` in this repo finds no other matches.

Reply:
I share your confusion.  It's here, simply [because it was also here.](https://github.com/kubernetes-incubator/cri-o/pull/1578/files) - the source material for "how to make it chooch".

Reply:
OIC, okay, this passes through ``make integration`` and down into ``test/test_runner.sh`` (according to other PR).